trillian: Add version 6.4.0.5

stasjok opened the pull request:
Trillian
https://www.trillian.im

Ash258 commented on bucket/trillian.json:
```diff
@@ -0,0 +1,41 @@
+    "url": "https://www.ceruleanstudios.com/trillian-v6.3.0.6.msi",
+    "hash": "f1ea8e70c5bad9380c95e29cc0f41159504f3c6a2fc38d6005f2c4ae985257f2",
+    "pre_install": [
+        "$file = 'trillian.ini'",
+        "if (!(Test-Path \"$persist_dir\\$file\")) {",
+        "   Write-Host 'File' $file 'does not exists. Creating.' -f Yellow",
+        "   $CONT = @(",
+        "       '[General]'",
+        "       'Use User Directory=0'",
+        "       'Global Directory=users\\global\\'",
+        "   )",
+        "   Set-Content \"$dir\\$file\" ($CONT -join \"`r`n\") -Encoding Ascii",
+        "}"
```
```suggestion
        "if (!(Test-Path \"$persist_dir\\trillian.ini\")) {",
        "   Set-Content \"$dir\\trillian.ini\" ('[General]', 'Use User Directory=0', 'Global Directory=users\\global\\') -Encoding Ascii",
        "}"
```

Ash258 commented on bucket/trillian.json:
```diff
@@ -0,0 +1,41 @@
+    ],
+    "checkver": {
+        "url": "https://www.trillian.im/shared/scripts/news.items.js",
+        "regex": "\"windows\",\\s*([\\d.]+),\\s*([\\d]+),\\s*\"[^\"]+\",\\s+false",
```
```suggestion
        "regex": "\"windows\",\\s*([\\d.]+),\\s*([\\d]+),\\s*\".*?\",\\s*false",
```

Ash258 reviewed (changes requested):
Blocked by lukesampson/scoop#3828

rashil2000 commented:
The latest URL is https://trillian.im/get/windows/6.5/trillian-v6.5.0.18.exe, but 7z cannot extract it. @issaclin32 does it work for you?

issaclin32 commented:
> The latest URL is https://trillian.im/get/windows/6.5/trillian-v6.5.0.18.exe, but 7z cannot extract it. @issaclin32 does it work for you?

Cannot extract either. (using 7-zip 21.07)

stasjok commented:
I've updated autoupdate URL.

rashil2000 commented:
Have you tested the manifest by installing it?

stasjok commented:
> Have you tested the manifest by installing it?

I've updated version that was already installed for me on my computer with this manifest. Looks like versions 6.4 and earlier are downloaded from ceruleanstudios.com, but since 6.5 they moved to trillian.im. So autoupdate was broken. But thanks for your message I noticed it and fixed for now.

rashil2000 commented:
/verify
